src/bower/DependenciesManager.js

```diff
@@ -77,10 +77,41 @@ define(function (require, exports) {
         return FileUtils.exists(path);
     }
 
+    /**
+     * Get the dir of the dependencies
+     * @param {string=} path
+     * @return {Promise}
+     */
+    function findBowerJsonLocation(path) {
+        if (!path) {
+            path = ProjectManager.getProjectRoot().fullPath;
+        }
+
+        return path;
+    }
+
     function _loadBowerJson(path) {
         _bowerJsonFile = new BowerJsonFile(path);
     }
 
+    function getDependencies() {
+        var deferred = new $.Deferred();
+
+        _bowerJsonFile.getContent()
+            .done(function (data) {
```

Comment:
I don't think so:). The getContent gets all the content of bower.json. This one also can be used for getting the content if you want to make it configurable. Within the getDependencies I get the dependencies from the content and create a result.

That's why it's called getContent